Packages/com.unity.inputsystem/InputSystem/Plugins/Android/AndroidSupport.cs

```diff
@@ -39,7 +46,68 @@ public static void Initialize()
 }
             ");
 
-            InputSystem.onFindTemplateForDevice += AndroidGameController.OnFindTemplateForDevice;
+            InputSystem.RegisterProcessor<AndroidAccelerationProcessor>();
+
+            // Add sensors
+            InputSystem.RegisterTemplate<AndroidAccelerometer>();
+            InputSystem.RegisterTemplate<AndroidMagneticField>();
+            InputSystem.RegisterTemplate<AndroidOrientation>();
+            InputSystem.RegisterTemplate<AndroidGyroscope>();
+            InputSystem.RegisterTemplate<AndroidLight>();
+            InputSystem.RegisterTemplate<AndroidPressure>();
+            InputSystem.RegisterTemplate<AndroidProximity>();
+            InputSystem.RegisterTemplate<AndroidTemperature>();
+            InputSystem.RegisterTemplate<AndroidGravity>();
+            InputSystem.RegisterTemplate<AndroidLinearAcceleration>();
+            InputSystem.RegisterTemplate<AndroidRotationVector>();
+            InputSystem.RegisterTemplate<AndroidRelativeHumidity>();
+            InputSystem.RegisterTemplate<AndroidAmbientTemperature>();
+            InputSystem.RegisterTemplate<AndroidMagneticFieldUncalibrated>();
+            InputSystem.RegisterTemplate<AndroidGameRotationVector>();
+            InputSystem.RegisterTemplate<AndroidGyroscopeUncalibrated>();
+            InputSystem.RegisterTemplate<AndroidSignificantMotion>();
+            InputSystem.RegisterTemplate<AndroidStepDetector>();
+            InputSystem.RegisterTemplate<AndroidStepCounter>();
+            InputSystem.RegisterTemplate<AndroidGeomagneticRotationVector>();
+            InputSystem.RegisterTemplate<AndroidHeartRate>();
+
+            InputSystem.onFindTemplateForDevice += OnFindTemplateForDevice;
+        }
+
+        internal static string OnFindTemplateForDevice(int deviceId, ref InputDeviceDescription description,
```

Comment:
Some naming changes have hit the repo yesterday. The concept of a "template" has been renamed to "control layout". Several APIs have been renamed. Please update to latest head.